cmd/exo/cmd/vm.go

```diff
@@ -11,11 +11,23 @@ var vmCmd = &cobra.Command{
 	Short: "Virtual machines management",
 }
 
-func getVMWithNameOrID(cs *egoscale.Client, name string) (*egoscale.VirtualMachine, error) {
+// nolint
+const (
+	starting = "Starting"
+	running  = "Running"
+	stoping  = "Stopping"
+	stopped  = "Stopped"
+)
```

Comment:
Is it worth creating it in egoscale then?

https://github.com/apache/cloudstack/blob/893a88d225276e45f12f9490e6af2c94a81c2965/api/src/main/java/com/cloud/vm/VirtualMachine.java#L46-L55

Reply:
I put it in egoscale is better